packages/react-native-reanimated/src/initializers.ts

```diff
@@ -7,11 +7,16 @@ import {
   registerLoggerConfig,
   SHOULD_BE_USE_WEB,
 } from './common';
+import { initSvgCssSupport } from './css/svg';
+import { EXPERIMENTAL_CSS_ANIMATIONS_FOR_SVG_COMPONENTS } from './featureFlags/staticFlags.json';
```

Comment:
As we discussed offline, we should replace the usage of the .json file with the `getStaticFlag` method, which will be available with this PR. https://github.com/software-mansion/react-native-reanimated/pull/8096
It can be done in another PR.